README.md

```diff
@@ -74,9 +74,8 @@ If you want to compile it from source, you will need Go 1.7 or later, and the [G
 3. Install dependencies using Glide:  
   `glide install`
 4. Compile the binary:  
-  - Linux/macOS/*nix: `go build -o akamai`
-  - Windows `go build -o akamai.exe`
-5. Move the binary (`akamai` or `akamai.exe`) it to your `PATH`
+  - Linux/macOS/*nix: `go install -o $GOPATH/bin/akamai .``
+  - Windows `go build -o $GOPATH/bin/akamai.exe`
```

Comment:
Would this not be `%GOPATH%` on windows? I don't currently have a Windows box to test on…